src/blog/2024/03/node-red-vpn-flowfuse.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Installing and operating Node-RED with a VPN
+subtitle: FlowFuse was built to empower Node-RED to run everywhere, also in a VPN
+description: FlowFuse was built to empower Node-RED to run everywhere, also in a VPN
+date: 2024-03-06
+authors: ["zeger-jan-van-de-weg"]
+image: # /blog/2024/03/images/node-red-vpn.png
+tags:
+    - posts
+    - flowfuse
+    - node-red
+    - vpn
+---
+
+Practitioners using Node-RED often find themselves in a situation where a VPN ([Virtual Private Network](https://en.wikipedia.org/wiki/Virtual_private_network)) is deployed in their organization. The VPN is a fact of life and is generally not controlled by the same people using Node-RED. Given security regions supreme in Industrial IoT (IIoT), and a VPN offers a lot of benefits, the VPN will be deployed more often instead of less, and as such it’s good to understand how you can get the most out of Node-RED when deployed in a VPN.
+
+## Node-RED installation with a VPN
+
+Generally the normal install procedure for Node-RED requires a connection being made to the NPM servers that host the package. Due to the unaudited nature of NPM, it’s unlikely that IT will agree with a permanent exception for the VPN to have access to it. There’s a couple of actions one can take to install Node-RED anyway.
```

Comment:
```suggestion
Generally, the normal install procedure for Node-RED requires a connection to be made to the NPM servers that host the package. Due to the unaudited nature of NPM, it’s unlikely that IT will agree with a permanent exception for the VPN to have access to it. There are a couple of actions one can take to install Node-RED anyway.
```